### PR TITLE
[patch] Patch added to rename net_weight field

### PR DESCRIPTION
**Pull request by vishdha**

### CODE REVIEW

File: erpnext/patches/v9_2/rename_net_weight_in_item_master.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from __future__ import unicode_literals
+import frappe
+from frappe.model.utils.rename_field import rename_field
+
+def execute():
+	frappe.reload_doc("stock", "doctype", "item")
+	if frappe.db.has_column('Item', 'net_weight'):
+		rename_field("Item", "net_weight", "weight_per_unit")
```
Comment (rohitwaghchaure) [inline erpnext/patches/v9_2/rename_net_weight_in_item_master.py]: Patch will break if field weight_per_unit is exists!

Comment (rohitwaghchaure) [reply erpnext/patches/v9_2/rename_net_weight_in_item_master.py]: if exists copy data from net_weight to weight_per_unit else rename_field("Item", "net_weight", "weight_per_unit")

**Comment by joezsweet** [comment]
What's the goal of this?

**Comment by vishdha** [comment]
Rename field net_weight to weight_per_unit in Item Master.

**Comment by joezsweet** [comment]
Got it, i mean what's the goal of renaming?

**Comment by vishdha** [comment]
We added shipping rule based on Net Weight. so, selling and buying module fetch weight data from the item master and calculate shipping charges based on that weight.

**Comment by joezsweet** [comment]
@vishdha ok, thx for explanation

**Pull request merged**